runtime/near-vm-errors/src/lib.rs

```diff
@@ -17,10 +17,33 @@ pub enum VMError {
     CacheError(CacheError),
 }
 
-#[derive(
-    Debug, Clone, PartialEq, Eq, BorshDeserialize, BorshSerialize, Deserialize, Serialize, RpcError,
-)]
+// TODO: remove serialization derives, once fix compilation caching.
+#[derive(Debug, Clone, PartialEq, Eq, BorshDeserialize, BorshSerialize, Serialize, Deserialize)]
 pub enum FunctionCallError {
```

Comment:
is this enum still needed given the presence of `FunctionCallErrorSer` and `ContractCallError`

Reply:
Yes, it's used inside the contract runtime and have precise execution problem description. We don't want it to leak beyond contract runtime.